mava/wrappers/jumanji.py

```diff
@@ -123,68 +129,85 @@ def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
 class ConnectorWrapper(MultiAgentWrapper):
     """Multi-agent wrapper for the MA Connector environment."""
 
-    def __init__(self, env: MaConnector):
+    def __init__(self, env: MaConnector, has_global_state: bool = True):
```

Comment:
This should default to false as we do in other wrappers

```suggestion
    def __init__(self, env: MaConnector, has_global_state: bool = False):
```